src/test/unit/engine/NetworkFactoryTest.cpp

```diff
@@ -30,15 +30,23 @@
 #include <nupic/engine/Network.hpp>
 #include <nupic/engine/NuPIC.hpp>
 #include <nupic/engine/Region.hpp>
+#include <nupic/os/Path.hpp>
 #include <nupic/utils/Log.hpp>
 
 using namespace nupic;
 
+// macro to ease nesting
+#define GP Path::getParent
+
+const std::string UNIT_TESTS_EXECUTABLE_PATH = Path::getExecutablePath();
+const std::string NUPIC_CORE_PATH =  GP(GP(GP(GP(UNIT_TESTS_EXECUTABLE_PATH))));
+const std::string PATH_TO_FIXTURES = Path::join(NUPIC_CORE_PATH, "/src/test/unit/engine/fixtures");
```

Comment:
I don't quite follow your comment but it seems like it will be more clear to remove the "/" prefix for the second string being joined.